### PR TITLE
Fix DataGrid and TreeList React demos (XXX prop -> defaultXXX prop)

### PR DESCRIPTION
**Pull request by jtoming830**

### CODE REVIEW

**Review by NickMitrokhin — DISMISSED** [review]

**Review by NickMitrokhin — DISMISSED** [review]

File: JSDemos/Demos/DataGrid/KeyboardNavigation/React/App.js [hunk 1]
```diff
@@ -8,26 +16,24 @@ class App extends React.Component {
...
+          allowUpdating={true}
+          allowDeleting={true}
+          selectTextOnEditStart={true}
+          useIcons={true}
+        />
+        <HeaderFilter visible={true} />
+        <FilterPanel visible={true} />
+        <FilterRow visible={true} />
+        <Pager
+          allowedPageSizes={[5, 10]}
```
Comment (vconst) [inline JSDemos/Demos/DataGrid/KeyboardNavigation/React/App.js]: Move this array to constant

File: JSDemos/Demos/DataGrid/RemoteVirtualScrolling/React/App.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ class App extends React.Component {
         wordWrapEnabled={true}
       >
         <Scrolling mode="virtual" rowRenderingMode="virtual" />
-        <Paging pageSize="100" />
+        <Paging defaultPageSize="100" />
```
Comment (vconst) [inline JSDemos/Demos/DataGrid/RemoteVirtualScrolling/React/App.js]: defaultPageSize={100}

File: JSDemos/Demos/TreeList/KeyboardNavigation/React/App.js [hunk 3]
```diff
@@ -10,39 +18,44 @@ class App extends React.Component {
...
+        <FilterPanel
+          visible={true}
+        />
+        <FilterRow
+          visible={true}
+        />
+        <Scrolling
+          mode='standard'
+        />
+        <Column dataField='Full_Name' />
```
Comment (vconst) [inline JSDemos/Demos/TreeList/KeyboardNavigation/React/App.js]: Use double quotes in React. Linter should show this.

File: JSDemos/Demos/TreeList/KeyboardNavigation/React/App.js [hunk 4]
```diff
@@ -10,39 +18,44 @@ class App extends React.Component {
...
+        <Column
+          dataField='Title'
+          caption='Position'
+        />
+        <Column dataField='City' />
+        <Column dataField= 'State' />
+        <Pager
+          allowedPageSizes={[5, 10]}
```
Comment (vconst) [inline JSDemos/Demos/TreeList/KeyboardNavigation/React/App.js]: Move allowedPageSizes to constant

**Review by vconst — CHANGES_REQUESTED** [review]

**Review by vconst — APPROVED** [review]

**Review by NickMitrokhin — APPROVED** [review]

**Pull request merged**